src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryTable.cs

```diff
@@ -4,6 +4,7 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Security.Cryptography.X509Certificates;
```

Comment:
>System.Security.Cryptography.X509Certificates; [](http://example.com/codeflow?start=6&length=46)

curious what we need this for?

Reply:
I think it got pulled in by VS with a incorrect autocomplete suggestion. I'll remove.